presto-main/src/main/java/com/facebook/presto/server/StatementResource.java

```diff
@@ -573,11 +573,11 @@ private synchronized URI createNextResultsUri(UriInfo uriInfo)
 
         private static List<Column> createColumnsList(QueryInfo queryInfo)
         {
-            requireNonNull(queryInfo, "queryInfo is null");
```

Comment:
There's no need for it, this is a private method that is always called with non-null